back/src/main/java/com/qmate/api/EmailVerificationController.java

```diff
@@ -15,16 +19,25 @@
 @RestController
 @RequestMapping("/auth/email-verifications")
 @RequiredArgsConstructor
+@Tag(name = "Email Verification", description = "이메일 인증 관련 API")
 public class EmailVerificationController {
   private final EmailVerificationService emailVerificationService;
 
   @PostMapping
+  @Operation(
+      summary = "이메일 인증 전송",
+      description = EmailVerificationConstants.SEND_MD
+  )
   public ResponseEntity<EmailSentResponse> send(@Valid @RequestBody EmailVerificationSendRequest req) {
     emailVerificationService.sendCode(req.getEmail(), req.getPurpose());
     return ResponseEntity.ok(new EmailSentResponse(true));
   }
 
   @PostMapping("/resend")
+  @Operation(
+      summary = "이메일 인증 재전송",
+      description = EmailVerificationConstants.RESEND_MD
+  )
   public ResponseEntity<?> resend(@Valid @RequestBody EmailVerificationSendRequest req) {
```

Comment:
```suggestion
  public ResponseEntity<EmailResentResponse> resend(@Valid @RequestBody EmailVerificationSendRequest req) {
```
와일드 카드 사용하면 스웨거문서에 response가 자동으로 매핑되지 않는것 같습니다 이부분 수정이 필요할것 같아요

Reply:
수정했습니다!